common/scylla_configure.py

```diff
@@ -90,7 +90,7 @@ def save_scylla_yaml(self):
             scylla_yaml_file.write(dedent(f"""
                 # Generated by Scylla Machine Image at {now}
                 # See '/etc/scylla/scylla.yaml.example' with the full list of supported configuration
-                # options and their descriptions."""[1:]))
+                # options and their descriptions.\n"""[1:]))
```

Comment:
This issue will make the first option be commented.
In our test, `api_address: 127.0.0.1` was commented, and the private ip was listened unexpectedly.

This commit fixed the issue :-), and new test passed.